examples/fuse_examples/multimodality/image_clinical/multimodality_image_clinical.ipynb

```diff
@@ -532,22 +238,22 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 8,
+   "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
     "\n",
-    "from fuse.dl.models.model_default import ModelMultiHead\n",
+    "from fuse.dl.models.model_multihead import ModelMultiHead\n",
     "from fuse.dl.models.heads.head_global_pooling_classifier import HeadGlobalPoolingClassifier\n",
     "from fuse.dl.models.backbones.backbone_inception_resnet_v2 import BackboneInceptionResnetV2\n",
     "from fuse.dl.models.backbones.backbone_resnet import BackboneResnet\n",
     "\n",
     "model = ModelMultiHead(\n",
-    "    conv_inputs=(('data.input.image', 3),),\n",
+    "    conv_inputs=(('data.input.img', 3),),\n",
     "    backbone=BackboneInceptionResnetV2(input_channels_num=3, pretrained_weights_url=None),\n",
     "    # backbone=BackboneResnet(in_channels=3),\n",
```

Comment:
remove comment
and "# tutorial todo" comment after this code block (not shown in the diff)